src/specializations/specializations.resolver.ts

```diff
@@ -0,0 +1,18 @@
+import { Resolver, Query, Mutation, Args } from '@nestjs/graphql';
+import { CreateSpecializationInput } from './dto/create-specialization.input';
+import { SpecializationsService } from './specializations.service';
+
+@Resolver('Specialization')
```

Comment:
why queries and mutations defined in schema but not in resolver ?

Reply:
I left it for future implementation